climpred/metrics.py

```diff
@@ -126,14 +134,84 @@ def _mae(forecast, reference, dim='svd', comparison=None):
     return mae(forecast, reference, dim=dim)
 
 
-def _crps(forecast, reference, dim='svd', comparison=None):
+def _brier_score(forecast, reference, **kwargs):
+    """Calculate Brier score for forecasts on binary reference.
+
+    ..math:
+        BS(f, o) = (f - o)^2
+
+    Args:
+        * forecast (xr.object)
+        * reference (xr.object)
+        * func (function): function to be applied to reference and forecasts
+                           and then mean('member') to get forecasts and
+                           reference in interval [0,1]  (required)
+
+    Reference:
+        * Brier, Glenn W. “VERIFICATION OF FORECASTS EXPRESSED IN TERMS OF
+        PROBABILITY.” Monthly Weather Review 78, no. 1 (1950).
+        https://doi.org/10.1175/1520-0493(1950)078<0001:VOFEIT>2.0.CO;2.
+
+    See also:
+        * properscoring.brier_score
+        * xskillscore.brier_score
+    """
+    if 'func' in kwargs:
```

Comment:
Again, this should all be deleted and just make `func` an explicit required argument.

Reply:
I would not like to implement it this way. because then the compute functions will have an endless list of arguments to be documented. this now is more implicit and requires the user to look into the metrics directly.

Reply:
I agree with @bradyrx on this so it would show up in the args/kwargs line, and can be autocompleted (I think?)

Reply:
I want to have all the logic of a metric inside the metric. would you prefer to have this check on args in the main compute_functions? I dont.

Reply:
I see the issue here. The argument is being passed on the `compute_perfect_model` level, for instance. Okay.